PsfLauncher/README.md

```diff
@@ -0,0 +1,99 @@
+
+## Configuration
+PSF Launcher uses a config.json file to configure the behavior.
+
+Given an application package with an `AppxManifest.xml` file containing the following:
+
+```xml
+<Package ...>
+  ...
+  <Applications>
+    <Application Id="PrimaryApp"
+                 Executable="PrimaryApp.exe"
+                 EntryPoint="Windows.FullTrustApplication">
+      ...
+    </Application>
+  <Applications>
+    <Application Id="PSFSample"
+                 Executable="PSFLauncher64.exe"
+                 EntryPoint="Windows.FullTrustApplication">
+      ...
+    </Application>
+  </Applications>
+</Package>
+```
+
+A possible `config.json` example would be:
+
+
+```json
+{
+	"applications": [
+	{
+		"id": "PSFLAUNCHERSixFour",
+      		"executable": "PrimaryApp.exe",
+      		"workingDirectory": "",
+      		"monitor": {
+			"executable": "PsfMonitor.exe",
+			"arguments": "",
+			"asadmin": true,
+        		"wait": false
+      		}
+    	}
+  	],
+	"processes": [
+	{
+		"executable": ".*PrimaryApp.*",
+		"shims": [ 
+		{ 
+			"dll": "TraceFixup64.dll",
+			"config": {
+					"traceMethod": "eventlog",
+					"traceLevels": {
+						"default": "always"
+					}
+			}
+		} 
+	        ]
+    	}
+  	]
+}
+```
+
+In this example, the configuration is directing the PsfLauncher to start PsfMonitor and then the referenced Primary App. PsfMonitor to be run using RunAs, followed by PrimaryApp once the monitoring app is stable. The root folder of the package is for the CurrentDirectory.  In the processes section of the example, the json further configures Trace Fixup to be injected into the Primary app, and that is to capture all levels of trace to the event log.
+
+
+| Array | key | Value |
+|-------|-----------|-------|
+| applications | id |  Use the value of the `Id` attribute of the `Application` element in the package manifest. |
+| applications | executable | The package-relative path to the executable that you want to start. In most cases, you can get this value from your package manifest file before you modify it. It's the value of the `Executable` attribute of the `Application` element. |
+| applications | workingDirectory | (Optional) A package-relative path to use as the working directory of the application that starts. If you don't set this value, the operating system uses the `System32` directory as the application's working directory. If you supply a value in the form of an empty string, it will use the directory of the referenced executable. |
+| applications | monitor | (Optional) If present, the monitor identifies a secondary program that is to be launched prior to starting the primary application.  A good example might be `PsfMonitor.exe`.  The monitor configuration consists of the following items:
+	`'executable'` - This is the name of the executable relative to the root of the package.
+	`'arguments'`  - This is a string containing any command line arguments that the monitor executable requires.

```

Comment:
What arguments can be specified/supported here for the PSFMonitor?

Reply:
PSFMonitor has no command line arguments at this time. I have updated the PSFMonitor Readme to reflect this.  The intent of arguments here is to support potential future changes in PSFMonitor, as well as supporting any third party tool that might be appropriate for this purpose.

Reply:
I would recommend keeping things simple, if there is no use for it today, lets remove it.

Reply:
In the launcher readme file please separate the monitor example from the simple original one.
The original example is good for start to explain how to configure the PSF.
The example with the monitor/trace is a bit more advanced and better to have it separately.
Also, there are couple of typos there such as: the app id is not matching "PrimaryApp" vs "PSFLAUNCHERSixFour" and shims have to be replaced with fixups.